src/main/java/spoon/support/compiler/SpoonProgress.java

```diff
@@ -0,0 +1,38 @@
+package spoon.support.compiler;
+
+/**
+ * is the interface to follow the progress of the creation of the Spoon model.
+ */
+public interface SpoonProgress {
+
+	enum Process {
+		COMPILE,
+		COMMENT,
+		MODEL,
+		IMPORT,
+		PROCESS,
+		PRINT
```

Comment:
I don't see in the diff the usage of PRINT? Shouldn't it used in DJPP?

Reply:
Done